jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -211,42 +212,91 @@ def get_regions(self, region_type=None, list_of_subset_labels=None,
+            Controls return type of ``PixelRegion`` / ``SkyRegion``. To return a spatial
+            subset in opposition with the current link type (e.g return ``PixelRegion``
+            when WCS linked, ``SkyRegion`` when pixel linked), ``wrt_data`` can be set to
+            the data label of the dataset whose WCS should be used for this transformation.
+            The default behavior (None) will return Pixel/Sky region based on app link type
+            (Sky for Cubeviz), using the WCS of the subset's parent dataset (i.e the data
+            layer the subset was created on).
 
         Returns
         -------
         regions : dict
             A dictionary mapping subset labels to their respective ``regions``
             objects (for spatial regions) or ``SpectralRegions`` objects
             (for spectral regions).
+
+        Examples
+        --------
+        >>> from jdaviz import Imviz, Cubeviz
+        >>> from regions import PixCoord, CirclePixelRegion, CircleSkyRegion
+        >>> from astropy.nddata import NDData
+        >>> import numpy as np
+        >>> import astropy.units as u
+        >>> imviz = Imviz()
+        >>> imviz.link_data(align_by='pixels')
+        >>> data = NDData(np.ones((128, 128)) * u.nJy, wcs=getfixture('image_2d_wcs'))
+        >>> imviz.load_data(data)
+        >>> plg = imviz.plugins['Subset Tools']
+        >>> plg.import_region(CirclePixelRegion(center=PixCoord(x=1163.618408203125, y=1433.47998046875), radius=141.28575134277344))  # noqa E501
+        >>> type(plg.get_regions()['Subset 1'])
+        <class 'regions.shapes.circle.CirclePixelRegion'>
+        >>> type(plg.get_regions(wrt_data='NDData[DATA]')['Subset 1'])
+        <class 'regions.shapes.circle.CircleSkyRegion'>
+        >>> imviz.app.delete_subsets()
+        >>> imviz.link_data(align_by='wcs')
+        >>> plg.import_region(CirclePixelRegion(center=PixCoord(x=1163.618408203125, y=1433.47998046875), radius=141.28575134277344))  # noqa E501
+        >>> type(plg.get_regions()['Subset 2'])
+        <class 'regions.shapes.circle.CircleSkyRegion'>
+        >>> type(plg.get_regions(wrt_data='NDData[DATA]')['Subset 2'])
+        <class 'regions.shapes.circle.CirclePixelRegion'>
+
+        >>> cubeviz = Cubeviz()
+        >>> cubeviz.load_data(getfixture('spectrum1d_cube'))
+        >>> plg = cubeviz.plugins['Subset Tools']
+        >>> plg.import_region(CirclePixelRegion(center=PixCoord(x=24.27156066879736, y=22.183517455582475), radius=4.7523674964904785))  # noqa E501
+        >>> type(plg.get_regions()['Subset 1'])
+        <class 'regions.shapes.circle.CircleSkyRegion'>
+        >>> type(plg.get_regions(wrt_data='Unknown spectrum object[FLUX]')['Subset 1'])
+        <class 'regions.shapes.circle.CirclePixelRegion'>
         """
+        if return_sky_region is not None:
+            raise ValueError('return_sky_region no longer used, use wrt_data instead')
```

Comment:
since this isn't using a deprecated decorator, is there some ticket to remove the kwarg entirely in a future release?  Also, how much effort do you think it would be (probably as a follow-up) to just automatically map `return_sky_region` to `wrt_data` during the deprecation period?

Reply:
It would just take some time to discuss what to do if `return_sky_region` and `wrt` conflict. Probably a small diff though.

Reply:
ok, is that here (should we discuss it now) or in one of the follow-up tickets?

Reply:
Added a small block to allow `return_sky_region` to work as before, let me know if that looks correct.